cypress/local-only/tests/integration/caseDetail/docketRecord/multidocket-qc-and-edit.cy.ts

```diff
@@ -0,0 +1,319 @@
+
+      cy.visit('/document-qc/section/inbox/selectedSection?section=docket');
+      cy.get(
+        `[data-testid="work-item-${consolidatedGroupInfo.leadDocketNumber}"]`,
+      ).should('exist');
+
+      cy.get(
+        `[data-testid="work-item-${consolidatedGroupInfo.leadDocketNumber}"]`,
+      )
+        .filter(':contains("Administrative Record")')
+        .find('[data-testid="checkbox-assign-work-item"]')
+        .click();
+      cy.get('[data-testid="dropdown-select-assignee"]').select(
+        'Test caseServicesSupervisor1',
+      );
+
+      cy.visit('/document-qc/my/inbox');
+      cy.get('[data-testid="case-link"]')
+        .contains(consolidatedGroupInfo.leadDocketNumber)
+        .should('have.length', 1);
+      cy.contains('a', 'Administrative Record').should('exist');
+    });
+
+    it('should show needs QC star on all docket entries in the consolidated group', () => {
+      loginAsDocketClerk1();
+
+      goToCase(consolidatedGroupInfo.leadDocketNumber);
+      cy.get('svg.fa-star.fa-icon-red').should('have.length.greaterThan', 0);
+
+      consolidatedGroupInfo.memberDocketNumbers.forEach(memberDocketNumber => {
+        goToCase(memberDocketNumber);
+        cy.get('svg.fa-star.fa-icon-red').should('have.length.greaterThan', 0);
+      });
+    });
+
+    it('should NOT display Complete QC link on member cases when document needs QC', () => {
+      loginAsDocketClerk1();
+
+      consolidatedGroupInfo.memberDocketNumbers.forEach(memberDocketNumber => {
+        goToCase(memberDocketNumber);
+
+        cy.get('[data-testid="complete-qc-button"]').should('not.exist');
+      });
+    });
+
+    it('should complete QC on all docket entries in all cases of the consolidated group when completing QC on lead case', () => {
+      loginAsCaseServicesSupervisor('caseServicesSupervisor1@example.com');
+      cy.visit('/document-qc/section/inbox/selectedSection?section=docket');
+
+      cy.get(
+        `[data-testid="work-item-document-link-${consolidatedGroupInfo.leadDocketNumber}"]`,
+      )
+        .contains('Administrative Record')
+        .click();
+
+      cy.get('#save-and-finish').click();
+
+      cy.get('[data-testid="success-alert"]').should('contain', 'QC Completed');
+
+      loginAsDocketClerk1();
+      goToCase(consolidatedGroupInfo.leadDocketNumber);
+      cy.get('[data-testid="document-viewer-link-ADMR"]')
+        .closest('tr')
+        .find('.filing-type-icon')
+        .should('be.empty');
+
+      consolidatedGroupInfo.memberDocketNumbers.forEach(memberDocketNumber => {
+        goToCase(memberDocketNumber);
+        cy.get('[data-testid="document-viewer-link-ADMR"]')
+          .closest('tr')
+          .find('.filing-type-icon')
+          .should('be.empty');
+      });
+    });
+  });
+
+  describe('Edit Docket Entry Process', () => {
+    describe('Lead Case', () => {
+      it('should allow editing a multidocketed docket entry in the lead case and display help text', () => {
```

Comment:
These are very thorough tests, thank you! Usually we try to keep each test independent of others, but in this case the set up is extensive enough that it's probably worth making an exception. It _might_ be worth adding a comment for future debuggers, that each describe block relies on the previous one having been run